conn/sticky_linux.go

```diff
@@ -0,0 +1,121 @@
+/* SPDX-License-Identifier: MIT
+ *
+ * Copyright (C) 2017-2022 WireGuard LLC. All Rights Reserved.
+ */
+
+package conn
+
+import (
+	"net/netip"
+	"unsafe"
+
+	"golang.org/x/sys/unix"
+)
+
+// TODO: macOS, FreeBSD and other BSDs likely do support this feature set, but
```

Comment:
This is a _linux.go file, so this comment seems out of place. Maybe put it in _default, or get rid of it all together.

Reply:
Moved to `sticky_default.go` in https://github.com/WireGuard/wireguard-go/pull/64/commits/2449a1d8e11dc3753c2a7990709f729e5937f325.